<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportEtabs</class>
 <widget class="QDialog" name="ExportEtabs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>505</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>9</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Export to ETABS</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>3</y>
     <width>830</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Choose .e2k File</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="2">
       <widget class="QPushButton" name="output_button">
        <property name="text">
         <string>Brows</string>
        </property>
        <property name="iconSize">
         <size>
          <width>256</width>
          <height>256</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Output</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="input_button">
        <property name="text">
         <string>Brows</string>
        </property>
        <property name="iconSize">
         <size>
          <width>256</width>
          <height>256</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="input_path_line"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="output_path_line"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>input</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>59</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>150</y>
     <width>831</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Grid Dimensions (Plan)</string>
      </property>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>34</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Uniform &amp;Grid Spacing</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_6">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>204</y>
         <width>143</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>C&amp;ustom Grid Spacing</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>256</x>
         <y>194</y>
         <width>16</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>351</width>
         <height>212</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Spacing of Grids in X Direction</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Spacing of Grids in Y Direction</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="typical_height_spinbox_5">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="suffix">
           <string>  m</string>
          </property>
          <property name="maximum">
           <double>99.000000000000000</double>
          </property>
          <property name="value">
           <double>3.600000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="typical_height_spinbox_4">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="suffix">
           <string>  m</string>
          </property>
          <property name="maximum">
           <double>99.000000000000000</double>
          </property>
          <property name="value">
           <double>3.600000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_3">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_15">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Specify Data for Grid Lines</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Number of Grid Lines in Y Direction</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_4">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Number of Grid Lines in X Direction</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QToolButton" name="toolButton_4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edit Grid Data</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Story Dimensions</string>
      </property>
      <widget class="QRadioButton" name="simple_story_radiobutton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>34</y>
         <width>137</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Simp&amp;le Story Data</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="custom_story_radiobutton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>203</y>
         <width>143</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Cus&amp;tom Story Data</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>256</x>
         <y>194</y>
         <width>16</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>271</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Number of Stories</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_10">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Specify Custom Story Data</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Bottom Story Height</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Typical Story Height</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="typical_height_spinbox">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;واحد طول باید بر مبنای واحد فایل ورودی وارد گردد.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="suffix">
           <string> m</string>
          </property>
          <property name="maximum">
           <double>20000.000000000000000</double>
          </property>
          <property name="value">
           <double>3.600000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="buttom_height_spinbox">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;واحد طول باید بر مبنای واحد فایل ورودی وارد گردد.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="suffix">
           <string> m</string>
          </property>
          <property name="maximum">
           <double>20000.000000000000000</double>
          </property>
          <property name="value">
           <double>3.600000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="number_of_story_spinox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QToolButton" name="toolButton_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edit Story Data</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>470</y>
     <width>815</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>input_button</tabstop>
  <tabstop>output_button</tabstop>
  <tabstop>number_of_story_spinox</tabstop>
  <tabstop>typical_height_spinbox</tabstop>
  <tabstop>buttom_height_spinbox</tabstop>
  <tabstop>input_path_line</tabstop>
  <tabstop>output_path_line</tabstop>
  <tabstop>spinBox_4</tabstop>
  <tabstop>toolButton_4</tabstop>
  <tabstop>custom_story_radiobutton</tabstop>
  <tabstop>typical_height_spinbox_5</tabstop>
  <tabstop>simple_story_radiobutton</tabstop>
  <tabstop>toolButton_3</tabstop>
  <tabstop>radioButton_6</tabstop>
  <tabstop>radioButton_5</tabstop>
  <tabstop>typical_height_spinbox_4</tabstop>
  <tabstop>spinBox_3</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>custom_story_radiobutton</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButton_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>591</y>
    </hint>
    <hint type="destinationlabel">
     <x>758</x>
     <y>657</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simple_story_radiobutton</sender>
   <signal>toggled(bool)</signal>
   <receiver>number_of_story_spinox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>758</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simple_story_radiobutton</sender>
   <signal>toggled(bool)</signal>
   <receiver>typical_height_spinbox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>758</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simple_story_radiobutton</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttom_height_spinbox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>758</x>
     <y>517</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_6</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButton_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>658</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_5</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_5</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_5</sender>
   <signal>toggled(bool)</signal>
   <receiver>typical_height_spinbox_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>517</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_5</sender>
   <signal>toggled(bool)</signal>
   <receiver>typical_height_spinbox_5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_5</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_14</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_5</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_19</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_5</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_17</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_5</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_16</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>567</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_6</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_15</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>604</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>640</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_story_radiobutton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_10</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>607</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>639</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simple_story_radiobutton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simple_story_radiobutton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simple_story_radiobutton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_6</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportEtabs</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>775</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>811</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportEtabs</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>772</y>
    </hint>
    <hint type="destinationlabel">
     <x>624</x>
     <y>807</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
